pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -148,11 +148,11 @@ def test_cython_agg_return_dict():
 
 def test_cython_fail_agg():
     dr = bdate_range("1/1/2000", periods=50)
-    ts = Series(["A", "B", "C", "D", "E"] * 10, index=dr)
+    ts = Series(["A", "B", "C", "D", "E"] * 10, dtype=object, index=dr)
 
     grouped = ts.groupby(lambda x: x.month)
     summed = grouped.sum()
-    expected = grouped.agg(np.sum)
+    expected = grouped.agg(np.sum).astype(object)
```

Comment:
Was there a specific reason you added an explicit `dtype=object` here (since it seems you only added this in the last commit, after updating for `sum()` being implemented, so now this is actually no longer needed, I think) ?